### ISSUE TITLE
Theme support?

### ISSUE THREAD

**Issue by bnssoftware** [surjithctly/astroship#55]
Do you have any plans to offer a light/dark theme switcher?

**Comment by surjithctly** [comment]
yes, in the meantime check this: https://github.com/surjithctly/astro-color-scheme
